projects/dex-ui/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "dex-ui",
+  "description": "A user interface for Beanstalk Dex",
+  "private": true,
+  "version": "1.0.0",
+  "license": "MIT",
+  "scripts": {
+    "start": "vite --port 2424",
+    "build": "vite build",
+    "lint": "npx eslint src",
```

Comment:
why run through npx, these will be installed locally anyway

Reply:
Will fix